spec/factories/organisation.rb

```diff
@@ -2,7 +2,25 @@
   factory :organisation do
     skip_create
 
+    transient do
+      providers { [] }
+      users { [] }
+    end
+
     sequence(:id)
     name { "Organisation" }
+    nctl_ids { [] }
+
+    after :build do |organisation, evaluator|
```

Comment:
Can we add the comment from the other factories about how this unexpected bit of code is necessary for the JSONAPI client?